### PR TITLE
updating type descriptions to use official language

### LINKED ISSUE #330: Types: Open
**Is your feature request related to a problem? Please describe.**
The existing "Type: Open" makes a specific Open Meeting declaration: This meeting is open and may attend.

**Describe the solution you'd like**
Local members say that the actual declaration should be same as it is listed in the Big Book as well as on the Meeting Guide App.

**Describe alternatives you've considered**
Change the declaration to: "Open meetings are available to any one interested in alcoholics anonymous program of recovery from alcoholism non-alcoholics may attend open meetings as observers."
Screen Shot 2021-05-04 at 3 01 07 PM

Should be: 

IMG_7A49DD03C1D4-1

**Additional context**

### PR DESCRIPTION
**Pull request by joshreisner**
i'm not such a fan of these descriptions, but they wore me down. i made this change in the meeting guide app last year and TSML UI last month, so it makes sense to do it here too.

to fix #330 

closed
open

### CODE REVIEW

**Review by tech2serve — APPROVED** [review]

**Comment by Oclair** [comment]
The message as currently is in place detracts from the meeting information page, too wordy and detailed and does not conform to a suggestive program but rather an authoritative directive from special workers in a publishing company.  I would suggest keeping this but adding an additional open meeting definition without the message.

The definition of what open and closed may have been recently defined and thus directed to AA autonomous groups to adhere to. But AA the founders did not make such a definition of what an open meeting should or should not be.  This should be left to AA affiliated autonomous groups to decide what is right for them in a group conscience. 

Thank you for your service
Jason D aavienna.com
